docs/run/agent/installation.md

```diff
@@ -1,4 +1,19 @@
-# Fleet Installation
+# Installation
+To get started with the agent, create a new cluster in the Insights UI, e.g. `staging`
```

Comment:
probably link to create cluster?

Reply:
I don't think we can, since the URL includes their org name